src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -459,7 +459,7 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
     }
     if (this.isCommunityContribution) {
       originalName = originalName.replace(' LTD.', '')
-      originalName += ' COMMUNITY CONTRIBUTION COMPANY'
+      originalName += ' COMMUNITY CONTRIBUTION COMPANY LTD.'
```

Comment:
I checked bcrs-shared-components and business-create-ui and no changes are needed there.